src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/x4decoder/common/FilteringErrorGroup.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2022 AVI-SPL, Inc. All Rights Reserved.
+ */
+package com.avispl.symphony.dal.avdevices.encoderdecoder.haivision.x4decoder.common;
+
+/**
+ * Set of Filtering Errors Group
+ *
+ * @author Harry
+ * @since 1.0
+ */
+public enum FilteringErrorGroup {
+
+	STREAM_NAME("Mismatch Stream Name Filter"),
```

Comment:
please move the error message to the constant instead of enum like th is

Reply:
maybe the DecoderConstant is the good candidate

Reply:
I got it. But it is not in use anymore, I will delete it